packages/autorest.typescript/src/models/operationDetails.ts

```diff
@@ -77,6 +77,7 @@ export interface OperationDetails {
   lroOptions?: { "final-state-via": string };
   scope?: Scope;
   namePrefix?: string;
+  isTenantLevel?: boolean;
```

Comment:
I didn't see a lot of value of operation-level property, what's the value of this?